R/gh_fill.R

```diff
@@ -0,0 +1,28 @@
+#' Fill geohash prefix with members
+#'
+#' @param geohashes Character vector of input geohashes. They must all be of same precision
+#' @param precision Positive integer scalar controlling the 'zoom level' – how many characters should be used in the output.
+#' @return Character vector of geohashes corresponding to the input.
+#' @export
+
+
+
+gh_fill <- function(geohashes, precision) {
+  if (uniqueN(nchar(geohashes)) > 1) {
```

Comment:
uniqueN is a data.table function, which we don't import. `length(unique())` should be fine here.